assets/stylesheets/shared/_forms.scss

```diff
@@ -211,7 +211,7 @@ select {
 	outline: 0;
 	overflow: hidden;
 	font-size: 14px;
-	font-weight: 600;
+	font-weight: bold;
```

Comment:
According to the [Design Handbook](https://wordpress.com/design-handbook/typography/), we should use numbers rather than names to ensure the weight is consistent